### PR TITLE
Update RxPermissions.java

### PR DESCRIPTION
**Pull request by Jadyli**
fix  a bug

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -37,21 +37,23 @@ ext {
...
-    gitUrl = 'https://github.com/tbruyelle/RxPermissions.git'
+    siteUrl = 'https://github.com/Sanqis/RxPermissions'
+    gitUrl = 'https://github.com/Sanqis/RxPermissions.git'
 
-    developerId = 'tbruyelle'
-    developerName = 'Thomas Bruyelle'
-    developerEmail = 'thomas.bruyelle@gmail.com'
+    developerId = 'Sanqis'
+    developerName = 'Jady Li'
+    developerEmail = '1257984872@qq.com'
```
Comment (T12E) [inline build.gradle]: Really ?

**Comment by tbruyelle** [comment]
Why did you change the author ids ?

**Comment by Jadyli** [comment]
because i need to gemerate a new version of rxpermission for use，but i don't know whether the develop id affects when i upload my  package to jack.io,i'm sorry for that ,later i will fix it

**Comment by Jadyli** [comment]
i don't know why the build.gradle was commit to here ，i only want to commit the Rxpermissions.java

**Comment by tbruyelle** [comment]
Please create a clean PR with the only the changes you want. I close this one for now.

**Pull request closed without merging**